samples/bluetooth/nrf_dm/src/main.c

```diff
@@ -316,7 +315,7 @@ static int bt_sync_init(void)
 
 	printk("DM Bluetooth LE Synchronization initialization\n");
 
-	mfg_data.company_code = sys_cpu_to_le16(COMPANY_CODE);
+	mfg_data.company_code = sys_cpu_to_le16(CONFIG_BT_COMPANY_ID);
```

Comment:
Isn't this Nordic Specific feature? I think this should be:
```suggestion
	mfg_data.company_code = sys_cpu_to_le16(BT_COMPANY_ID_NORDIC);
```

Reply:
I don't know mesh that well :p